### ISSUE TITLE
Broken audio on wasapi

### ISSUE THREAD

**Issue by AccountForStealingCode** [mpv-player/mpv#13384]
### Important Information

- mpv v0.37.0-229-gc6ccabe5 (shinchiro 20240128)
- Windows 10 LTSC x64 22H2

### Reproduction steps

Play audio

### Expected behavior

Normal audio

### Actual behavior

Noisy broken audio

### Log file

[--ao=wasapi](https://github.com/mpv-player/mpv/files/14120548/wasapi.txt)
[--ao=openal (working)](https://github.com/mpv-player/mpv/files/14120549/openal.txt)

### Sample files

https://0x0.st/HD9d.flac

**Comment by AccountForStealingCode** [comment]
i tried again and openal, sdl plays it fine
wasapi (+wmp) is still broken
also tried foobar2000 and it also works

**Comment by Raza-0007** [comment]
> i tried again and openal, sdl plays it fine
wasapi (+wmp) is still broken

I just tried your second sample and it plays identically on wasapi, openal, and sdl on my system (Win 11). 

When you say broken, what is it your are hearing? There was a noise in the beginning of the clip on all three, but when the girl starts talking everything is crystal clear.

Are you using updated audio drivers? I was having issues with opus audio not performing well on my system. Windows update said I was using the latest audio drivers, even though my drivers were from 2019! I had to actually manually search for the latest drivers on MS catalog using my HW ID. Updating audio drivers resolved my issue.

~Raza
